src/azure-cli/requirements.py3.windows-spython.txt

```diff
@@ -0,0 +1,139 @@
+azure-core==1.24.0
+azure-cosmos==3.2.0
+azure-data-tables==12.4.0
+azure-datalake-store==0.0.49
+azure-graphrbac==0.60.0
+azure-keyvault-administration==4.0.0b3
+azure-keyvault-keys==4.5.1
+azure-keyvault==1.1.0
+azure-loganalytics==0.1.0
+azure-mgmt-advisor==9.0.0
+azure-mgmt-apimanagement==3.0.0
+azure-mgmt-appconfiguration==2.1.0
+azure-mgmt-applicationinsights==1.0.0
+azure-mgmt-authorization==0.61.0
+azure-mgmt-batch==16.2.0
+azure-mgmt-batchai==7.0.0b1
+azure-mgmt-billing==6.0.0
+azure-mgmt-botservice==2.0.0b3
+azure-mgmt-cdn==12.0.0
+azure-mgmt-cognitiveservices==13.2.0
+azure-mgmt-compute==27.1.0
+azure-mgmt-consumption==2.0.0
+azure-mgmt-containerinstance==9.1.0
+azure-mgmt-containerregistry==10.0.0
+azure-mgmt-containerservice==20.2.0
+azure-mgmt-core==1.3.1
+azure-mgmt-cosmosdb==7.0.0b6
+azure-mgmt-databoxedge==1.0.0
+azure-mgmt-datalake-analytics==0.2.1
+azure-mgmt-datalake-nspkg==3.0.1
+azure-mgmt-datalake-store==0.5.0
+azure-mgmt-datamigration==10.0.0
+azure-mgmt-deploymentmanager==0.2.0
+azure-mgmt-devtestlabs==4.0.0
+azure-mgmt-dns==8.0.0
+azure-mgmt-eventgrid==10.2.0b2
+azure-mgmt-eventhub==10.1.0
+azure-mgmt-extendedlocation==1.0.0b2
+azure-mgmt-hdinsight==9.0.0
+azure-mgmt-imagebuilder==1.1.0
+azure-mgmt-iotcentral==10.0.0b1
+azure-mgmt-iothub==2.2.0
+azure-mgmt-iothubprovisioningservices==1.1.0
+azure-mgmt-keyvault==10.1.0
+azure-mgmt-kusto==0.3.0
+azure-mgmt-loganalytics==13.0.0b4
+azure-mgmt-managementgroups==1.0.0
+azure-mgmt-maps==2.0.0
+azure-mgmt-marketplaceordering==1.1.0
+azure-mgmt-media==9.0.0
+azure-mgmt-monitor==4.0.1
+azure-mgmt-msi==6.0.1
+azure-mgmt-netapp==8.1.0
+azure-mgmt-network==20.0.0
+azure-mgmt-policyinsights==1.1.0b2
+azure-mgmt-privatedns==1.0.0
+azure-mgmt-rdbms==10.2.0b3
+azure-mgmt-recoveryservices==2.1.0
+azure-mgmt-recoveryservicesbackup==5.0.0
+azure-mgmt-redhatopenshift==1.1.0
+azure-mgmt-redis==13.1.0
+azure-mgmt-relay==0.1.0
+azure-mgmt-reservations==2.0.0
+azure-mgmt-resource==21.1.0b1
+azure-mgmt-search==8.0.0
+azure-mgmt-security==2.0.0b1
+azure-mgmt-servicebus==7.1.0
+azure-mgmt-servicefabric==1.0.0
+azure-mgmt-servicefabricmanagedclusters==1.0.0
+azure-mgmt-servicelinker==1.0.0
+azure-mgmt-signalr==1.1.0
+azure-mgmt-sql==4.0.0b2
+azure-mgmt-sqlvirtualmachine==1.0.0b3
+azure-mgmt-storage==20.1.0
+azure-mgmt-synapse==2.1.0b2
+azure-mgmt-trafficmanager==1.0.0
+azure-mgmt-web==7.0.0
+azure-multiapi-storage==0.9.0
+azure-nspkg==3.0.2
+azure-storage-common==1.4.2
+azure-synapse-accesscontrol==0.5.0
+azure-synapse-artifacts==0.13.0
+azure-synapse-managedprivateendpoints==0.3.0
+azure-synapse-spark==0.2.0
+bcrypt==3.2.0
+certifi==2021.10.8
+cffi==1.15.0
+chardet==3.0.4
+colorama==0.4.4
+cryptography==3.3.2
+fabric==2.4.0
+humanfriendly==10.0
+idna==2.8
+invoke==1.2.0
+isodate==0.6.0
+javaproperties==0.5.1
+Jinja2==3.0.3
+jmespath==0.9.5
+jsondiff==2.0.0
+knack==0.10.0
+MarkupSafe==2.0.1
+msal-extensions==1.0.0
+msal==1.18.0b1
+msrest==0.7.0
+msrestazure==0.6.4
+multidict==6.0.2
```

Comment:
Add it in case of requests is not found. https://github.com/Azure/azure-sdk-for-python/blob/azure-core_1.24.0/sdk/core/azure-core/azure/core/utils/_utils.py#L104

Reply:
If latest `azure-core` implements its own `case_insensitive_dict`, we should bump to latest `azure-core`.